net/net-memcache-server.cpp

```diff
@@ -756,10 +758,10 @@ int return_one_key_list (struct connection *c, const char *key, int key_len, int
   ptr += 14;
   if (res != 0x7fffffff) {
     if (mode >= 0) {
-      w = sprintf (ptr, "%d", res);
+      w = snprintf(ptr, ptr_size - 14, "%d", res);
```

Comment:
why `ptr_size - 14`?
It's actually 498 when "%d" is maximum 12 bytes including null terminated symbol. Does it make sense?